README.md

```diff
@@ -117,6 +117,26 @@
   End each file with a newline.
   <sup>[[link](#newline-eof)]</sup>
 
+* <a name="trailing-commas"></a>
+  When dealing with lists, maps, structs or tuples whose elements span over multiple lines and are on separate lines with regard to the enclosing brackets, use a trailing comma even for the last element:
+  <sup>[[link](#trailing-commas)]</sup>
+
+  ```elixir
+  # Bad
+  [
+    :foo,
+    :bar,
+    :baz
+  ]
+
+  # Good
+  [
+    :foo,
+    :bar,
+    :baz,
```

Comment:
I'm enough in favour of that to convince every one, I'm not scared :) Easier reordering of items, less noisy diff when adding/removing items, really really weak arguments in favour of it. What's yours? :stuck_out_tongue: